pkg/prometheus/promcfg.go

```diff
@@ -4124,6 +4146,27 @@ func (cg *ConfigGenerator) generateScrapeConfig(
 	return cfg, nil
 }
 
+func (cg *ConfigGenerator) generateOTLPConfig(_ *assets.StoreBuilder) (yaml.MapItem, error) {
```

Comment:
If the Storebuilder isn't used, could we remove it from the arguments' list?

Reply:
+1

Reply:
Thanks. Got it. Done.